### ISSUE TITLE
Bromite Webview additional build variant

### ISSUE THREAD

**Issue by MichaelZ4714** [bromite/bromite#1889]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md)
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have searched [existing issues](https://github.com/bromite/bromite/issues) for my feature request. This is a new issue (NOT a duplicate) and is not related to another issue.

### Is your feature request related to privacy?

Yes

### Is there a patch available for this feature somewhere?

No

### Describe the solution you would like

I suggest to add another build variant to the Bromite Webview release. I think there is some demand for the apk having the package name as until v97. I understand that it was modified, but I see no chance for me to further use an up to date Bromite Webview, which would be an awful pity. If a second build of an apk with the previous package name and a titel such as Bromite Webview Classic or Bromite Webview Previous or whatever good name and this apk could be downloaded in parallel to the present variant from Github and would be distributed in parallel to the present variant through fdroid.bromite.org/fdroid/repo all people who cannot use the present variant would still be able to use a privacy friendly Webview. 

### Describe alternatives you have considered

The Magisk module WebviewManager does not work on my device.
I considered modifying framework-res.apk in /system/framework, but I failed.
I tried to change package name in BromiteWebview.apk, but I failed.

**Comment by csagan5** [comment]
There is only 1 official SystemWebView package, sorry. And it is provided "as is": you have to figure out how to use it, it is for advanced users.
